cats-effect/src/main/scala/com/avast/sst/catseffect/package.scala

```diff
@@ -0,0 +1,32 @@
+package com.avast.sst
+
+import java.util.concurrent.TimeUnit
+
+import cats.effect.syntax.bracket._
+import cats.effect.{Bracket, Clock}
+import cats.syntax.flatMap._
+import cats.syntax.functor._
+
+import scala.concurrent.duration.Duration
+
+package object catseffect {
+
+  /**  */
+  implicit class FOps[F[_], A](private val f: F[A]) extends AnyVal {
```

Comment:
I believe more standard way how to implement this is to move this code into `ClockHelper` (or whatever name), rename this package to include `syntax` name, and reference  `ClockHelper` from here.

Reply:
I am not sure where this particular method belongs so please do check the names and packages I created if it makes sense.